From d15b12ae923a565a25f1ad6eaa93ad65da2d0529 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Fri, 30 Jun 2023 16:52:13 +0200
Subject: [PATCH 314/841] refactor(st): move FWU metadata json file to common
 part

This file will be shared between all STM32MPU platforms.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 59d692f48165158dc2e1de5356a4440f1ea6efd8 refactor(st): move FWU metadata json file to common part]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I7d8f099ca64a8e0bfd277b3128739a1c7eb6e7d4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/315864
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
Reviewed-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
---
 plat/st/common/common.mk                           | 2 +-
 plat/st/{stm32mp1 => common}/default_metadata.json | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename plat/st/{stm32mp1 => common}/default_metadata.json (100%)

diff --git a/plat/st/common/common.mk b/plat/st/common/common.mk
index 84f3d5851..5079bfe72 100644
--- a/plat/st/common/common.mk
+++ b/plat/st/common/common.mk
@@ -44,7 +44,7 @@ ifeq (${PSA_FWU_SUPPORT},1)
 # Number of banks of updatable firmware
 NR_OF_FW_BANKS			:=	2
 NR_OF_IMAGES_IN_FW_BANK		:=	1
-JSON_METADATA			?=	plat/st/${PLAT}/default_metadata.json
+JSON_METADATA			?=	plat/st/common/default_metadata.json
 STM32_DEPS			+=	${BUILD_PLAT}/metadata.bin
 
 $(eval $(call GEN_METADATA,${JSON_METADATA},${BUILD_PLAT}/metadata.bin))
diff --git a/plat/st/stm32mp1/default_metadata.json b/plat/st/common/default_metadata.json
similarity index 100%
rename from plat/st/stm32mp1/default_metadata.json
rename to plat/st/common/default_metadata.json
-- 
2.47.2

